Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "curve-limit-order-bot-cw"
 version = "0.1.0"
-authors = ["stevenjung"]
+authors = ["Volume Fi"]
 edition = "2021"
```

Comment:
It's not 2021. Upgrade to 2024?

Reply:
Thank you. But I think the edition we are using is 2021.

Reply:
It was created automatically. I didn't want to make me the alone author.

Reply:
no worries. in future, you can just use the name volume